xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -561,6 +561,10 @@ void CGUIDialogPVRTimerSettings::OnSettingAction(const std::shared_ptr<const CSe
 
 bool CGUIDialogPVRTimerSettings::Validate()
 {
+  // Timer rules only have times, not dates, so we can't check those
```

Comment:
Hm, not sure this statement is actually correct. The concept allows for timer rules with dates. Why shouldn't it be possible to define a timer rule only firing from certain date+time until certain date+time?

Reply:
I guess you're right but then we should be able to distinguish between timers with only times and timers with time and dates, but I suspect there's currently no easy way to do that right?

Reply:
Right.